python/cvmfs/repository.py

```diff
@@ -292,10 +459,9 @@ def retrieve_certificate(self):
         return Certificate(certificate)
 
 
-    @abc.abstractmethod
     def retrieve_file(self, file_name):
-        """ Abstract method to retrieve a file from the repository """
-        pass
+        """ Method to retrieve a file from the repository """
```

Comment:
I don't agree with that. The user wants to download a file, and has a repository object. He doesn't need to know about the Fetcher object, and its structure, but just to tell the repository to download/fetch a file. The Fetcher object haven't been shown at any point to the user, why should we do it now?